docs/4_1_co-simulation_math.adoc

```diff
@@ -266,11 +266,13 @@ _The variables computed by_ latexmath:[\mathbf{f}_{\mathit{doStep}}] _can be inq
 
 ==== Smoothness, Continuity and Discontinuity
 
-Since values of variables will be set at specific communication points by the importing tool, the FMU must make assumptions about the values between these communication points.
+Since inputs will be set at specific communication points by the importing tool, the FMU must make assumptions about the values between these communication points, including points of intermediate updates.
 
-<<continuous, `Continuous`>> variables can be assumed to continue changing during these intervals according to smoothness and continuity assumptions (see <<recommendedIntermediateInputSmoothness>>).
-If a <<continuous>> variable changes `discontinuously` (the smoothness assumption fails at this communication point), the co-simulation algorithm must raise an event to indicate the discrete change to the FMU.
+Between communication points, even when intermediate updates are called, all changes must be assumed to be continuous.
 
-<<discrete, `Discrete`>> variables keep their values between communication points changing their values.
-Furthermore, changing <<discrete>> variables will likely require special handling within the FMU.
+<<continuous, `Continuous`>> inputs can be assumed to continue changing between communication points according to the extrapolation polynomial build with the output derivatives of the connected FMU (see <<recommendedIntermediateInputSmoothness>>).
```

Comment:
an extrapolation polynomial is only one possibility, the master can do other things.
I would replace 

> <<continuous, `Continuous`>> inputs can be assumed to continue changing between communication points according to the extrapolation polynomial build with the output derivatives of the connected FMU (see <<recommendedIntermediateInputSmoothness>>).
with 

<<continuous, `Continuous`>> inputs may change between communication in case of <<intermediateVariableSetRequested>>`== true`. These are provided by the the co-simulation algorithm for example by an extrapolation polynomial build with the output derivatives of the connected FMU (see <<recommendedIntermediateInputSmoothness>>).

Reply:
Sorry, I don´t understand the == true.
Could you please tell me how the first sentence ends?

Reply:
Oh, that disappeared at copy-paste. Was ment to be 
``
<<continuous, Continuous>> inputs may change between communication in case of <<intermediateVariableSetRequested >>`== true`. These are provided by the the co-simulation algorithm for example by an extrapolation polynomial build with the output derivatives of connecte FMUs (see <<recommendedIntermediateInputSmoothness>>).
``

Reply:
With minor corrections accepted in the text.